homeassistant/components/ipp/coordinator.py

```diff
@@ -50,9 +59,16 @@ def __init__(
             update_interval=SCAN_INTERVAL,
         )
 
-    async def _async_update_data(self) -> IPPPrinter:
+    async def _async_update_data(self) -> IPPDataUpdateCoordinatorType:
         """Fetch data from IPP."""
         try:
-            return await self.ipp.printer()
+            data = await self.ipp.printer()
         except IPPError as error:
             raise UpdateFailed(f"Invalid response from API: {error}") from error
+
+        if self.data is None or self.data["printer"].info.uptime > data.info.uptime:
```

Comment:
Do we add the variance check here as well?

Reply:
we don't need, since the scan interval is already 60s and the `info.uptime` is in "seconds since boot"